app/lib/logic/rolling_updates.rb

```diff
@@ -272,16 +279,16 @@ def missing_config
       end
 
       class Policies < Base
-        def missing_config
-          false
+        def enabled?
+          true
         end
-      end
 
-      class PolicyRegistry < Base
-        def enabled?
-          super && provider_can_use?(:policies)
+        def deprecated?
+          true
         end
+      end
 
+      class PolicyRegistry < Base
```

Comment:
We probably also need deprecated in this policy right?

Reply:
It is not being removed. Should it be removed?

Reply:
We do not remove the feature just will enforce it

Reply:
This is another rolling update and is out of scope. Usages are not removed so should not be deprecated. I suggest we merge this one and then work on the other one separately.